uavcan/pnp/65534.NodeIDAllocationData.1.0.uavcan

```diff
@@ -0,0 +1,99 @@
+#
```

Comment:
Why is it not possible to unite node ID allocation for CAN2.0 and other?

Reply:
Anonymous multi-frame transfers are not possible, they can be only single-frame. The maximum payload length of a CAN 2.0 single-frame transfer is limited to 7 bytes. In order to allocate a node ID, we have to communicate the unique ID of the local node (which is 16 bytes large) to the allocator. Seeing as it won't fit in one CAN frame, the allocatee and the allocator have to engage in a very complex multi-stage exchange process, which also has certain [known hard-to-fix pitfalls](https://uavcan.org/Specification/6._Application_level_functions/#duties-of-the-leader).

The CAN FD-based protocol is vastly simpler: the allocatee can just send the request in one message, and the receive the response immediately.

Reply:
That makes sense. Can we keep it in v0.1 just until the node allocator is tested and works with CAN-FD?